.travis.yml

```diff
@@ -7,11 +7,18 @@ cache:
   directories:
   - node_modules
 before_install:
-  - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.7.0
-  - export PATH=$HOME/.yarn/bin:$PATH
-  - yarn --version
```

Comment:
We don't need that, each time we run yarn it will output its version automatically.